support/system_cgroup/README

```diff
@@ -0,0 +1,21 @@
+01/30/2023 -- John Buck
+
+In order to support executing submissions in a sandbox on Linux, we use CGROUPs V2.  Therefore,
+the system you are running this on must support that.  Most newer Linux systems do.
```

Comment:
"the system you are running this on" seems very ambiguous.  What is the "this" that's being referred to?   It's not true, for example, that an Admin can't be run on Windows and still use CGroups on a Linux system for judging (just that the Admin can't "rejudge" a run).  I think the explanation of what is/isn't possible needs to be a little more robust.

Reply:
Changed to reflect that the systems that are performing judging must support cgroups v2.  Changed in[ this commit](https://github.com/pc2ccs/pc2v9/pull/732/commits/2b0672d8c485301faaf3960fd2f768146e52d4eb).